networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/client_nsx.py

```
@@ -236,6 +236,13 @@ def get_unique(self, path: str, params: dict = dict()) -> dict:
         elif results:
             return results
 
+    def search(self, path: str, params: dict) -> dict:
+        res = self.get(path=path, params=params)
+        if not res.ok:
+            #ToDo handle errors properly
+            pass
```

Should we at least raise an exception or what should happen then?